Main.js

```diff
@@ -2575,7 +2575,6 @@ function init_ui() {
 	init_zoom_buttons();
 	init_combat_tracker();
 
-	token_menu();

```

Comment:
Removed this out of main and made it a background task on main since someone can load in first. Will negligibly speed up load in